acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/factory/PantheonNodeFactory.java

```diff
@@ -120,6 +121,17 @@ public PantheonNode createArchiveNodeWithP2pDisabled(final String name) throws I
             .build());
   }
 
+  public PantheonNode createArchiveNodeWithP2pDisabledAndAdmin(final String name)
```

Comment:
Does it matter what type of node we are using (archive)? Can't we simplify the name to something like: `createNodeWithP2pDisabled`?

Reply:
👍